app/api/ada/isValidAdaAddress.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import { request } from './lib/request';
+
+export type IsValidAdaAddressParams = {
+  ca: string,
+  address: string,
+};
+
+export const isValidAdaAddress = (
+  { ca, address }: IsValidAdaAddressParams
+): Promise<boolean> => {
+  const encodedAddress = encodeURIComponent(address);
+  return request({
+    hostname: 'localhost',
+    method: 'GET',
+    path: `/api/addresses/${encodedAddress}`,
```

Comment:
This method does not work like "expected"? because the cardano api just returns empty string for invalid addresses (which actually produces a cryptic error in the request object because it doesn't handle empty strings correctly)!

Reply:
Just for the reference - this is a known Api bug: https://iohk.myjetbrains.com/youtrack/issue/CSM-557
...which was fixed in: https://github.com/input-output-hk/cardano-sl/pull/1885